lib/components/steps/mapping/MappingTab.tsx

```diff
@@ -100,6 +115,10 @@ const MappingTab = ({defaultCollection}: MappingProps) => {
     const [visibleRows, setVisibleRows] = useState<string[]>([]);
     const [selectableCollections, setSelectableCollections] = useState<SelectableCollection[]>([]);
     const [optionsMap, setOptionsMap] = useState<{ [id: string]: Option }>({});
+    const [currentFilterStrategy, setCurrentFilterStrategy] = useState<CurrentFilterInterface>();
+    const [variableNameSortOrder, setVariableNameSortOrder] = useState(0);
+    const [cdeSortOrder, setCdeSortOrder] = useState(0);
+    const [statusSortOrder, setStatusSortOrder] = useState(0);
```

Comment:
Issue (blocking): I was expecting to just have 1 state variable for the sortOrder. Every time we change the sorting filter we also reset the sorting order state variable